v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/utils/ElasticsearchIO.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 1260 Google LLC
```

Comment:
Since templates already uses 2.36 version should we directly consider using Beam ElasticSearch IO and drop the teleport version.  However this needs proper validation of all 3 existing templates